bin/inference/pycbc_inference_plot_gelman_rubin

```diff
@@ -29,9 +29,7 @@ from pycbc.inference import (gelman_rubin, io, option_utils)
 # add options to command line
 parser = io.ResultsArgumentParser(skip_args=['walkers'])
 
-# verbose option
```

Comment:
This script doesn't use init_logging (see approx line 56) and uses different definitions of each level (`--verbose` becomes DEBUG, not INFO). So we should either use init_logging or keep the current option

Reply:
For simplicity, I've reverted the change to that file in https://github.com/gwastro/pycbc/pull/4621/commits, we can always update it further down the line